common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -1082,8 +1082,7 @@ bool ap_wlan_hal_dwpal::update_vap_credentials(
     for (const auto &it : hostapd_config_vaps) {
         // Skip STAs
         std::string entry_mode;
-        if (hostapd_config_get_value(it.second, "mode", entry_mode) &&
-            entry_mode != "ap") {
+        if (hostapd_config_get_value(it.second, "mode", entry_mode) && entry_mode != "ap") {
```

Comment:
How is this related to the change?

Reply:
It doesn't. the Clang-format changed it.